pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/export/DataExporter.java

```diff
@@ -675,11 +675,17 @@ void enrichWithDSMEventDates(Handle handle,
                     ? participantProxyGuids.get(participantGuid).stream().findFirst().get()
                     : participantGuid;
             if (governancePolicy != null) {
-                AgeOfMajorityRule aomRule = governancePolicy.getApplicableAgeOfMajorityRule(handle,
-                        pexInterpreter,
-                        participantGuid,
-                        operatorGuid)
-                        .orElse(null);
+                AgeOfMajorityRule aomRule = null;
+                try {
+                    aomRule = governancePolicy.getApplicableAgeOfMajorityRule(handle,
+                            pexInterpreter,
+                            participantGuid,
+                            operatorGuid)
+                            .orElse(null);
+                } catch (Exception e) {
+                    log.warn("Error while evaluating age-of-majority rules for participant {} operator {} and studyId {}, ignoring.",
```

Comment:
wondered if log.error is better but ended up using warn to avoid unnecessary alerts in prod.
This is just misses adding dateOfMajority in elastic export for dsm to display for these ptps.
Actual AgeOfMajority process is not impacted with this change.

Reply:
I tried to avoid stacktrace. I will do it which helps